DAO.sol

```diff
@@ -826,32 +948,23 @@ contract DAO is DAOInterface, Token, TokenCreation {
 
 
     function changeAllowedRecipients(address _recipient, bool _allowed) noEther external returns (bool _success) {
-        if (msg.sender != curator)
+        if (msg.sender != curator && (msg.sender != address(this) || _allowed))
```

Comment:
This desperateley needs a comment.

So now the DAO can call `changeAllowedRecipient()` only to remove someone from the whitelist?